recipe/meta.yaml

```diff
@@ -10,7 +10,8 @@ source:
   sha256: 8eddc033e716f8c91c6a2112f0a8ebc5e00532b4a6ae1eb0ccc48e027f9c671c
 
 build:
-  number: 1
+  number: 0
```

Comment:
why reseting number from 1 to 0?

Reply:
This was my mistake. Then I rebuild it again and uploaded artifacts with the **build number 2**.
Now I'm preparing a linter script for checking the build number before building a package

Reply:
Why is this an issue? the version was bumped from `22.2.1` to `22.3.0` according to the full diff of the PR, so resetting the build-number to 0 looks reasonable?!